chainer/functions/pooling/roi_average_align_2d.py

```diff
@@ -66,15 +68,15 @@ def _get_bilinear_interp_params(y, x, y_low, x_low, y_high, x_high):
         // empty
         return false;
     }
-    if (p <= 0) {
-        p = 0;
-    }
     low = (int)p;
```

Comment:
`(int)p` doesn't work when p is negative.
https://ideone.com/Hle9w3

Reply:
actually this is because of  `floor`.
Thank you I update to use `floor`.
```
(int)(-2.5) # -2.0
(int)floor(-2.5) # -3.0
```